### ISSUE TITLE
uui-card-media image not aligned

### ISSUE THREAD

**Issue by acoumb** [umbraco/Umbraco.UI#430]
### Which *exact* UI Library version are you using? For example: 1.0.1 - don't just write v1.

1.0.0

### Bug summary

Umbraco 10.3.1 instance and a `uui-card-media` control added to a package.

I am pulling record details from a DAM API and use the asset title and URL:

```
<uui-card-media name="{{vm.selectedRecord.title}}">
    <uui-action-bar slot="actions">
        <uui-button look="secondary" label="Remove" ng-click="vm.removeAsset()">Remove</uui-button>
    </uui-action-bar>
    <img src="{{vm.selectedRecord.uri}}" alt="{{vm.selectedRecord.title}}" />
</uui-card-media>
```

The rendered item looks like this, with the image not fitting:
![image](https://user-images.githubusercontent.com/95346674/221798096-aab62284-26d9-44d9-86ac-5915b4ec6d64.png)

I also noticed that the issue is present in the [storybook](https://uui.umbraco.com/?path=/docs/uui-card-media--aaa-overview) as well:
![image](https://user-images.githubusercontent.com/95346674/221798200-08daaafe-0a77-4ba2-b051-e1527900337d.png)

 

### Steps to reproduce

![image](https://user-images.githubusercontent.com/95346674/221798356-f3a1ed0d-d43c-49c9-8aca-249f8e495ee9.png)

**Comment by arknu** [comment]
I think I'm hitting the same thing, with this console error:
```
Uncaught TypeError: Cannot read properties of undefined (reading '0')
    at n.UUICardMediaElement.queryPreviews (index.js:31:25)
    at Ci.handleEvent (lit-html.js:6:7113)
queryPreviews @ index.js:31
```

**Comment by nathanwoulfe** [comment]
This should be fixed in v1.2.0, if you want to try installing the RC to verify.

**Comment by iOvergaard** [comment]
Fixed in v1.2.0
